### DISCUSSION TITLE
Time to revisit CDISC standards with the shift to modern data formats/db?

### DISCUSSION THREAD

**Discussion by lengning** [rinpharma/rinpharma-summit-2024#7] [roundtable]
### Proposal

A pressing topic I think we should discuss is......
...

### Expected impact

I believe this topic would be a benefit because......
...

### Prior discussions/work

Some prior work includes......
...

### Would you be willing to potentially facilitate this discussion?

None

**Comment by epijim** [comment]
+1 for this. It feels CDISC data was designed in a time before joins being trivial to execute, and being able to create things like views that pre-define that structure but present to users that easy to consume 'one row per patient' dataset. So we end up with very bloated data, and it's near impossible to convert some data types into CDISC without blowing up storage needs (as a dataset in relational format could be many multiples larger as ADaM data...)

One question maybe though is R/Pharma the place for this vs PHUSE (which is much closer to CDISC)?

**Reply by harveyl888** [reply]
Could certainly start a discussion at R/Pharma and then turn to PHUSE to turn it into a deliverable?

**Comment by epijim** [comment]
added to the agenda
